Python/bltinmodule.c

```diff
@@ -239,7 +239,9 @@ builtin___build_class__(PyObject *self, PyObject *const *args, Py_ssize_t nargs,
 }
 
 PyDoc_STRVAR(build_class_doc,
-"__build_class__(func, name, /, *bases, [metaclass], **kwds) -> class\n\
+"__build_class__(func, name, /, *bases, **kwds)\n\
+(func, name, /, *bases, metaclass, **kwds)\n\
```

Comment:
The builtin functions lack `$module`, no?

```suggestion
"__build_class__($module, func, name, /, *bases, **kwds)\n\
($module, func, name, /, *bases, metaclass, **kwds)\n\
```

Reply:
It is not required, isn't? `$module` is removed in any case.

Reply:
Sure, I just think it is good practice.